options.go

```diff
@@ -197,6 +197,7 @@ type Options struct {
 	Quality        int
 	Compression    int
 	Zoom           int
+	Fit            bool // Fit image into given dimensions
```

Comment:
I needed a new option to bypass settings `Force=true` in `normalizeOperation` later
I can't use `Crop` or `Embed`:
- `Crop` will crop extra pixels
- `Embed` will pad missing pixes with background